datarepo-clienttests/src/main/java/utils/FileUtils.java

```diff
@@ -128,7 +131,28 @@ public static String createGsPath(
 
     // save a reference to the JSON file so we can delete it in cleanup()
     createdScratchFiles.add(fileRefName);
-    return "gs://" + testConfigGetIngestbucket + "/" + fileRefName;
+    return String.format("gs://%s/%s", testConfigGetIngestbucket, fileRefName);
+  }
+
+  public static String createGcsPath( // TODO combine this and above method
```

Comment:
Agreed that combining this with the above method would be good code reuse. Perhaps you could write to a ByteArrayOutputStream or use a StringBuilder and then get the bytes to pass to the above method from that.